WordPress/src/main/java/org/wordpress/android/ui/people/PeopleListFragment.java

```diff
@@ -189,6 +179,15 @@ public void onBindViewHolder(PeopleAdapter.PeopleViewHolder holder, int position
             if (mOnFetchMorePeopleListener != null && position == getItemCount() - 1) {
                 mOnFetchMorePeopleListener.onFetchMorePeople();
             }
+
+            holder.itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    if (mOnPersonSelectedListener != null) {
```

Comment:
We handle click listeners in recyclers this same way in much of our code (and I'm responsible for some of them), but it turns it's not always safe - there are cases where relying on a `final` variable this way can result in the click listener using the wrong item.

A safer way is for the listener to use `getAdapterPosition()` to locate the position of the clicked view, and then get the model (a `Person` in this case) from that position. An example of this can be seen in our [CommentAdatper](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/people-infinite-scroll/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java#L102-L118).